lib/global_uid/configuration.rb

```diff
@@ -23,8 +23,8 @@ def initialize
       # Duration (in seconds) to wait before attempting another connection to UID server
       @connection_retry = 600 # 10 minutes
 
-      # This proc is called with two parameters upon UID server failure -- an exception and a message
-      @notifier = Proc.new { |exception, message| ActiveRecord::Base.logger.error("GlobalUID error: #{exception.class} #{message}") }
+      # This class is called with the exception on a UID server failure
```

Comment:
```suggestion
      # An object that will be notified in case of a UID server failure. The object is expected to respond to `#notify`, which will be passed one argument of type `Exception`.
```